pkg/planner/core/fragment.go

```diff
@@ -114,6 +114,9 @@ type mppTaskGenerator struct {
 	frags      []*Fragment
 	cache      map[int]tasksAndFrags
 
+	// fragMap maps ExchangeReceiver to its fragments
+	fragMap map[*physicalop.PhysicalExchangeReceiver][]*Fragment
```

Comment:
how this comes

Reply:
ai optimization?.....I do not know why it auto optimize it ...